samza-api/src/main/java/org/apache/samza/operators/functions/InitableFunction.java

```diff
@@ -24,16 +24,16 @@
 
 
 /**
- * interface defined to initalize the context of message transformation functions
+ * A function that can be initialized before execution.
  */
 @InterfaceStability.Unstable
 public interface InitableFunction {
 
   /**
-   * Interface method to initialize the context for a specific message transformation function.
+   * Initializes the function before any messages are processed.
    *
```

Comment:
Not currently. If there are multiple input streams, it'll depend on the iteration order for the Map that contains them.

That said, we should try to guarantee a deterministic and predictable iteration order if possible. Created SAMZA-1271 to track.